app/src/showcase.html

```diff
@@ -0,0 +1,230 @@
+      <ux-card sm="6">
+        <ux-card-header><span class="ux-card__title">Labels on top</span></ux-card-header>
+        <ux-card-content>
+          <ux-form>
+            <div class="ux-form__row">
+  
+              <ux-field label="First Name">
+                <ux-input variant.bind="variant" maxlength="10" value.bind="firstName & validate"></ux-input>
+                <ux-input-info ux-input-counter>John</ux-input-info>
+              </ux-field>
+  
+              <ux-field label="Last Name">
+                <ux-input variant.bind="variant" maxlength="10" value.bind="lastName & validate"></ux-input>
+                <ux-input-info ux-input-counter>Doe</ux-input-info>
+              </ux-field>
+  
+            </div>
+  
+            <ux-field label="Email Address">
+              <ux-input variant.bind="variant" type="email" value.bind="email & validate"></ux-input>
+              <ux-input-info>email@example.com</ux-input-info>
+            </ux-field>
+
+            <ux-field label="Contact Type">
+              <ux-select variant.bind="variant" value.bind="contactType" placeholder="Choose an option">
+                <ux-option>Private</ux-option>
+                <ux-option>Professionnal</ux-option>
+              </ux-select>
+            </ux-field>
+  
+            <ux-field label="Biography">
+              <ux-textarea variant.bind="variant" auto-resize></ux-textarea>
+            </ux-field>
+
+            <div class="ux-form__row">
+              <ux-field>
+                <label>Interests</label>
+                <div class="ux-form__row">
+                  <ux-checkbox checked.bind="interests" model.bind="'sport'"></ux-checkbox>
+                  <label>Sport</label>
+                </div>
+                <div class="ux-form__row">
+                  <ux-checkbox checked.bind="interests" model.bind="'technology'"></ux-checkbox>
+                  <label>Technology</label>
+                </div>
+                <div class="ux-form__row">
+                  <ux-checkbox checked.bind="interests" model.bind="'business'"></ux-checkbox>
+                  <label>Business</label>
+                </div>
+                <div class="ux-form__row">
+                  <ux-checkbox checked.bind="interests" model.bind="'travel'"></ux-checkbox>
+                  <label>Travel</label>
+                </div>
+                <div class="ux-form__row">
+                  <ux-checkbox checked.bind="interests" model.bind="'art'"></ux-checkbox>
+                  <label>Art</label>
+                </div>
+                <div class="ux-form__row">
+                  <ux-checkbox checked.bind="interests" model.bind="'politics'"></ux-checkbox>
+                  <label>Politics</label>
+                </div>
+              </ux-field>
+
+              <ux-field>
+                <label>Age Group</label>
+                <div class="ux-form__row">
+                  <ux-radio checked.bind="ageGroup" model.bind="'0-10'"></ux-radio>
```

Comment:
for demo purpose, shouldn't these radios have the same name?

Reply:
Yes, this is right. It works thanks to the shared bindable but for markup exemple we should add the name.

Reply:
@bigopon 
Fix: https://github.com/platform5/ux/commit/a3b14eefb37514c49087f051fd01841fc170528a